nym-wallet/src/theme/theme.tsx

```diff
@@ -43,12 +43,16 @@ const darkMode: NymPaletteVariant = {
   topNav: {
     background: '#111826',
   },
+  nav: {
+    background: '#FFFFFF',
+    hover: '#',
```

Comment:
This is a strange value to set - does `hover` maybe need to be made optional?

Reply:
At this moment `hover` is not used at all, should be safe to remove it.